### PR TITLE
Handle file not found error

### LINKED ISSUE #6: Failure if there is no entity in training data
cross_validation_results.py line number 42 expects the DIETClassifier_report.json to be present. It fails we there is no entity in training data. Ideally there should be exception handling to verify the presence of this file.

`with open(f"{result_dir}/DIETClassifier_report.json", "r") as f:`

### PR DESCRIPTION
**Pull request by HotThoughts**
Resolves #6

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -176,6 +176,5 @@ jobs:
         # https://github.com/RasaHQ/rasa-demo/blob/main/.github/workflows/continuous_integration.yml#L199-L207
         test_core_args: >-
           --stories tests/test_conversations.yml
-          --fail-on-prediction-errors
```
Comment (tczekajlo) [inline .github/workflows/ci.yml:179]: Why did you remove it?

Comment (HotThoughts) [reply .github/workflows/ci.yml:179]: The `--fail-on-prediction-errors` flag is used in the rasa-demo so I included it also in this action. As I working on patches, I found that the presence of prediction errors is a random thing. I often need to re-run the workflow till no more prediction errors. And yes, we can keep the flag if this is part of the feature we want to test.

Comment (HotThoughts) [reply .github/workflows/ci.yml:179]: What do you think?

**Review by tczekajlo — APPROVED** [review]
LGTM 🎖️
